### ISSUE TITLE
Order: Change ID for all operations to spSubscriptionId

### ISSUE THREAD

**Issue by junetaw** [on-api/on-api#77]
Note: This excludes "Activate a service" of course.

In 2.5.0 the required ID to deactivate, change, abuse block/unblock etc is subscriptionId. Example:
{
  "operation": "DEACTIVATE",
  "subscriptionId": "35738e19ab534dff9f9becb3a064a7d5",
  "requestedDateTime": "2019-02-05T00:00:00Z"
}

This ID is set by the CO and is not controlled by the SP.

In many instances, this ID might change. 
The CO may:
- do migrations internally
- manually remove and then add a service (for testing purposes)
- export the service to a new CO in a CO Change process

In all of the above cases, the subscriptionId is likely to change leaving the SP with incorrect data.

In the newly released Beta version of the CO Change process, spRef and spSubscriptionId should be migrated from a CO to another. Similarly the CO must maintain these values in any internal changes they make.

I therefore propose that subscriptionId is changed to be spSubscriptionId in ON-api 2.6.
This will also of course mean that spSubscriptionId must be mandatory and if a CO manually creates a service this must either be queried from the SP or the SP must be notified that they need to MODIFY the subscription.

There could also be a dual ID option where SP could supply either subscriptionId or spSubscriptionId but then it might be easier (recommended?) to implement two different operations?

**Comment by teemufrosen** [comment]
Working group meeting: Won't do. Similar issues would arise from changing to SP generated id. Also, it's a process issue rather than API issue.
